data/hai missions.txt

```diff
@@ -13,6 +13,7 @@ mission "First Contact: Hai"
 	source
 		attributes "hai"
 	on offer
+		log "Made contact with a species north of human space, through a wormhole. They call themselves the Hai, and seem to be peaceful. But they warn of renegades in the north of their space."
```

Comment:
I'm currently drowning in condition-checking like that in my own mod, so I'm starting to think it might generally be a bad idea. I'm not quite prepared to give up on the idea, but maybe in this case we are just better not mentioning either way. Logs don't fit into conditional logic well yet, anyway. (That's something that'd help at some point, being able to stick them in conversation branches and whatnot.)